capa/features/extractors/dnfile/helpers.py

```diff
@@ -119,14 +119,14 @@ def get_dotnet_managed_imports(pe: dnfile.dnPE) -> Iterator[DnType]:
         access: Optional[str]
 
         # assume .NET imports starting with get_/set_ are used to access a property
-        if member_ref.Name.startswith("get_"):
+        if str(member_ref.Name).startswith("get_"):
```

Comment:
should we do conversions like this once (where repeated)?

Reply:
looks like it is repeated only in this function.  That's a simple fix.  I'll take care of it

Reply:
thanks!